adagrams/game.py

```diff
@@ -1,11 +1,110 @@
+from random import randint
+from copy import deepcopy
+
+LETTER_POOL = [
+                "A", "A", "A", "A", "A", "A", "A", "A", "A",
+                "B", "B",
+                "C", "C",
+                "D", "D", "D", "D", 
+                "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", 
+                "F", "F",
+                "G", "G", "G",
+                "H", "H",
+                "I", "I", "I", "I", "I", "I", "I", "I", "I", 
+                "J",
+                "K",
+                "L", "L", "L", "L", 
+                "M", "M",
+                "N", "N", "N", "N", "N", "N", 
+                "O", "O", "O", "O", "O", "O", "O", "O", 
+                "P", "P",
+                "Q",
+                "R", "R", "R", "R", "R", "R", 
+                "S", "S", "S", "S", 
+                "T", "T", "T", "T", "T", "T", 
+                "U", "U", "U", "U", 
+                "V", "V",
+                "W", "W",
+                "X",
+                "Y", "Y",
+                "Z"
+    ]
+
+
 def draw_letters():
-    pass
+    # create a copy of letter pool so we don't alter original list
+    letter_pool_copy = deepcopy(LETTER_POOL)
+    drawn_letters = []
+    
+    # until the length of drawn_letters is 10, keep generating
+    # a random index to pop off from the letter pool 
+    # and append popped element to drawn_letters
+    while len(drawn_letters) < 10:
+        rand_index = randint(0, len(letter_pool_copy)-1)
+        drawn_letters.append(letter_pool_copy.pop(rand_index))
```

Comment:
Interesting way to randomly pick a letter! I also love that you modeled your `LETTER_POOL` list to contain the correct frequency of letters in a bag. Great work!

Reply:
But also...what if the instructors were feeling ~ chaotic ~ and said there were 100,000 tiles in the pool in total? Could you have programmatically made the `LETTER_POOL` list? (I'm sure you could have!)